src/run_spineopt_mga.jl

```diff
@@ -0,0 +1,61 @@
+function rerun_spineopt!(
+    ::Nothing,
+    ::Nothing,
+    m::Model,

```

Comment:
If this is the mga case, we can name the argument `m_mga`so the reader knows immediately?